taxonomies/additives_classes.txt

```diff
@@ -364,7 +364,7 @@ da: Emulgator, Emulgatorer, Emulgeringsmiddel
 el: Γαλακτωματοποιητής
 es: Emulgente, Emulsificante, emulsionante, agente dispersante, agente enturbiador, agentes enturbiadores, agente tensoactivo, corrector de la densidad, estabilizadores de una suspensión, estabilizador de una suspensión, inhibidor de la cristalización, plastificante
 fi: Emulgointiaine, Emulgointiainetta, Emulgointiaineet, Emulgointiaineita
-fr: Émulsifiant, agent de dispersion, agent de surface, agent de suspension, agent d'ajustement de la densité, inhibiteur de cristallisation, nébulisant, plastifiant
+fr: Émulsifiant, émulsiﬁant, agent de dispersion, agent de surface, agent de suspension, agent d'ajustement de la densité, inhibiteur de cristallisation, nébulisant, plastifiant
```

Comment:
émulsiﬁant with a single fi character is incorrect, we will fix it through spellchecking.